src/passport-saml/saml.ts

```diff
@@ -673,10 +699,10 @@ class SAML {
       if (!Object.prototype.hasOwnProperty.call(doc, 'documentElement'))
         throw new Error('SAMLResponse is not valid base64-encoded XML');
 
-      inResponseTo = xmlCrypto.xpath(doc, "/*[local-name()='Response']/@InResponseTo");
+      const inResponseToNodes = xmlCrypto.xpath(doc, "/*[local-name()='Response']/@InResponseTo") as Node[];
```

Comment:
Why aren't the `xmlCrypto` types handling this case correctly?

Reply:
returning value depends on input string, typescript does not know about `/@InRenponseTo` response. Updated to `as Attr[]`